README.md

```diff
@@ -35,40 +36,59 @@ which maintains the project and offers professional services around it.
 - Small memory footprint and fast execution
 - Domain-driven design architecture, high-quality code and test coverage
 
-## Install
-Before proceeding, install the following prerequisites:
+## Prerequisites
+
+The following are needed to run Mainflux:
+
+- [Docker](https://docs.docker.com/install/) **(template version 3.7)**
```

Comment:
Templates are related to `docker-compose`, not to Docker (Engine),

Reply:
I'll move this to be next to docker-compose.

Searching through the code, I'm not seeing anywhere that a specific docker engine is specified.  I'm currently using 18.09 but I'm wondering if I should install the latest (19.03) and see if there are any issues.